src/models/Popup.js

```diff
@@ -452,6 +523,10 @@ export default class Popup extends Base {
     // returns the parent element with the specified class, or the original element - null if not found
     const btn = traverseDOMPath(event.target, 'cc-btn') || event.target
     
+    if (btn.classList.contains( 'cc-btn' ) && btn.classList.contains( 'cc-customize' )){
+      this.showCustomizeSettingsContent();
+      return
```

Comment:
As the early exit here is very important, please space it out to make it stand out.

```suggestion

      return;
```